Addition of Facility Location

Tsimmons1234 opened the pull request:
Added facility location to "Space" folder of "101" section design patterns.

Finn1928 commented:
The colors of cells and borders should be standardized with other submissions. Missing ID for Facility. And I think the | symbols should be removed. Lastly, I think this should be placed in the CCO Whitepaper folder instead of the 101 folder.

Tsimmons1234 commented:
Borders and cells now standardized with previous submissions. Added ID for term 'facility'. Files now placed in design-pattterns/cco-whitepapers/space. Removed from design-patterns/101/space.

Tsimmons1234 commented:
Fixed error where borders for classes were rounded. Now standardized with rest of submissions.

giacomodecolle commented:
Looking good, but I see ont00000192 as code for Facility.
